src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  padding: 0 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 4px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    margin: 32px auto 0;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__name {
+    margin-top: 40px;
+    font-family: Roboto, sans-serif;
```

Comment:
let's move this property to html to avoid code duplication

Reply:
Not fixed